src/com/facebook/buck/jvm/kotlin/KotlinLibraryDescription.java

```diff
@@ -193,8 +193,65 @@ public void findDepsForTargetFromConstructorArgs(
   }
 
   public interface CoreArg extends JavaLibraryDescription.CoreArg {
+    /**
+     * @deprecated use {@link #getFreeCompilerArgs()}.
+     */
+    @Deprecated
     ImmutableList<String> getExtraKotlincArguments();
 
+    ImmutableList<String> getFreeCompilerArgs();
+
+    @Value.Default
+    default boolean getAllWarningsAsErrors() {
+      return false;
+    }
+
+    @Value.Default
+    default boolean getSuppressWarnings() {
+      return false;
+    }
+
+    @Value.Default
+    default boolean getVerbose() {
+      return false;
```

Comment:
seems like verbose should be true by default as per the usage in below `KotlincStep`

Reply:
I think that was incorrect before, as the default is `false`